addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/messages/RFXComMessage.java

```diff
@@ -46,6 +47,15 @@
     byte[] decodeMessage() throws RFXComException;
 
     /**
+     * Procedure for converting RFXCOM value to openHAB command.
+     *
+     * @param valueSelector
+     *
+     * @return Openhab command.
+     */
+    Command convertToCommand(RFXComValueSelector valueSelector) throws RFXComException;
+
```

Comment:
Because a State cannot be INCREASE or DECREASE. They can only be in Commands. The core forces us to deal in two different things even if they are both derived from the same base and very _nearly_, but not quite, compaible.